### DISCUSSION TITLE
Server-side Supabase for auth

### DISCUSSION THREAD

**Discussion by ioucyf** [supabase/supabase#8784] [Questions]
Is there a proper way to use supabase on the server-side that'll allow us to use httpOnly cookies instead of using it on the client using localeStorage? 

Also, generally speaking, what's the recommended way of using supabase auth (along with the rest of its services)? And, what are the best practices to follow?

**Comment by jamalmoir** [comment]
I'm also wondering about this, did you find a solution?

I've been using server-side authentication, but I believe, as the httpOnly cookies are set on the supabase domain, they are not available on my domain.

**Comment by saltcod** [comment]
Hey all 

We've got some recently released auth-helpers here that might help point you in the right direction:
https://github.com/supabase/auth-helpers
